ts/store/reducers/__tests__/featureFlagWithMinAppVersionStatus.test.ts

```diff
@@ -0,0 +1,529 @@
+        backendStatus: store,
+        mainLocalFlag: true,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- eq min version & local flag disabled", () => {
+    const store = customConfigNativeLogin({
+      min_app_version: { ios: "2.0.0.0", android: "2.0.0.0" }
+    });
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: false,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- < min version & local flag disabled", () => {
+    const store = customConfigNativeLogin({
+      min_app_version: { ios: "1.0.0.0", android: "1.0.0.0" }
+    });
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: false,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- > min version & local flag disabled", () => {
+    const store = customConfigNativeLogin({
+      min_app_version: { ios: "3.0.0.0", android: "3.0.0.0" }
+    });
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: false,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- min version undefined & local flag disabled", () => {
+    const store = customConfigNativeLogin({});
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: false,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- min version undefined & local flag enabled", () => {
+    const store = customConfigNativeLogin({});
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: true,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- nativelogin undefined & local flag enabled", () => {
+    const store = O.some({
+      config: {}
+    }) as unknown as O.Option<BackendStatus>;
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: true,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+
+  it("should return (nativelogin) disabled --- nativelogin undefined & local flag disabled", () => {
+    const store = O.some({
+      config: {}
+    }) as unknown as O.Option<BackendStatus>;
+    expect(
+      isPropertyWithMinAppVersionEnabled({
+        backendStatus: store,
+        mainLocalFlag: false,
+        configPropertyName: "nativeLogin"
+      })
+    ).toBeFalsy();
+  });
+});
+
+describe("backend service Feature Flag with min app version --- with optional config", () => {
```

Comment:
What about putting a "fast login" reference, here too? I mean, in the main description?